### PR TITLE
Add troubleshooting for "Not enough permissions" error

### PR DESCRIPTION
**Pull request by superojla**
**Description**

Changes proposed in this pull request:

- Created a new troubleshooting document
- Added information what to do if you receive a "Not enough permissions" error

**Related issue(s)**

### CODE REVIEW

File: docs/security/10-04-not-enough-permissions.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: '"Not enough permissions" error'
+type: Troubleshooting
+---
+
+If you log in to the Kyma console and receive an error message `Not enough permissions`:
```
Comment (klaudiagrz) [inline docs/security/10-04-not-enough-permissions.md:6]:
```suggestion
If you log in to the Kyma Console and receive `Not enough permissions` error message:
```

File: docs/security/10-04-not-enough-permissions.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+1. Fetch the ID Token. For example, use the [Chrome Developer Tools](https://developers.google.com/web/tools/chrome-devtools) and search for the token in sent requests authorization header.
```
Comment (klaudiagrz) [inline docs/security/10-04-not-enough-permissions.md:8]:
```suggestion
1. Fetch the ID Token. For example, use [Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools) and search for the token in the sent requests authorization header.
```

File: docs/security/10-04-not-enough-permissions.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+2. Decode the ID Token. For example, use the [jwt.io](https://jwt.io/) page.
+3. Check the value of `"email_verified"` property:
```
Comment (klaudiagrz) [inline docs/security/10-04-not-enough-permissions.md:10]:
```suggestion
3. Check the value of the **"email_verified"** property:
```

File: docs/security/10-04-not-enough-permissions.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  "exp": 1595525592,
+  "iat": 1595496792,
+  "azp": "console",
+...
+  "email": "{YOUR_EMAIL_ADDRESS}",
+  "email_verified": false,
+}
+```
+
+4. If the value is `false`, it means that IDP was unable to verify your email address. Contact your IDP for further guidance.
```
Comment (klaudiagrz) [inline docs/security/10-04-not-enough-permissions.md:29]:
```suggestion
4. If the value is set to `false`, it means that the IDP was unable to verify your email address. Contact your IDP for further guidance.
```

Comment (klaudiagrz) [reply docs/security/10-04-not-enough-permissions.md:29]: Consider changing IDP to `identity provider`.

Comment (superojla) [reply docs/security/10-04-not-enough-permissions.md]: Changed.

**Review by klaudiagrz — APPROVED** [review]
LGTM 🚀

**Pull request merged**